.changes/next-release/feature-751c6024-4e83-46c2-bbd3-d6222d39afe8.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Amazon Q Code Transform: Improve download failure ux"
```

Comment:
Is it possible to add more description/elaborate on what ux is changing here?

Reply:
done!